types/google.visualization/index.d.ts

```diff
@@ -654,6 +672,11 @@ declare namespace google {
             reverseCategories?: boolean;
             selectionMode?: string // single / multiple
             series?: any;
+            domainAxis?: { type: string };
+            trendlines?: any;
+            pointShape?: any;
```

Comment:
According to [docs](https://developers.google.com/chart/interactive/docs/points), `pointShape` accepts `string` value like `circle` and `triangle` etc.

Reply:
Agreed, corrected